Cargo.toml

```diff
@@ -11,10 +11,19 @@ serde = "1.0"
 tokio = {version = "1.17", features = ["macros", "rt-multi-thread"]}
 futures = "0.3"
 rand = {version = "0.8", default-features = false}
+aes = "0.8"
+cbc = "0.1"
+hmac = "0.12"
+sha2 = "0.10"
+zeroize = "1.5"
 
 [[bin]]
 name = "basic"
 path = "examples/basic.rs"
 
 [dev-dependencies]
 serde_json = "1.0"
+
+[dependencies.cipher]
```

Comment:
I would inline it like the other dependencies above

Reply:
Done.